embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/internal/payload/EnvelopeResource.kt

```diff
@@ -1,104 +1,201 @@
+ * @param buildId A unique ID for the build that is generated at build time. Previous name: a.bi
+ * @param appEcosystemId Unique identifier for the app in its ecosystem.
+ * In Apple, this is the Bundle ID (e.g. com.io.embrace).
+ * In Android, this is the app's package name (e.g. io.embrace.testapp). Previous name: a.bid
+ * @param buildType (Android) - the buildType name. Previous name: a.bt
+ * @param buildFlavor (Android) - the flavor name. If productFlavors are not used this will be null. Previous name: a.fl
+ * @param environment The name of the environment, i.e. dev or prod. Previous name: a.e
+ * @param bundleVersion The app bundle version (on Android this is versionCode). Previous name: a.bv
+ * @param sdkVersion The version number of the Embrace SDK. Previous name: a.sdk
+ * @param sdkSimpleVersion The simple version number of the Embrace SDK. Previous name: a.sdc
+ * @param reactNativeBundleId (React Native) the MD5 hash of the React Native bundle file. Previous name: a.rn
+ * @param reactNativeVersion (React Native) the React Native version number. Previous name: a.rnv
+ * @param javascriptPatchNumber (React Native) the JavaScript patch number. Previous name: a.jsp
+ * @param hostedPlatformVersion The version of the hosted platform engine, i.e.
+ * Unity/React Native/Flutter. Previous name: a.unv
+ * @param hostedSdkVersion The version of the hosted SDK used. Previous name: a.usv
+ * @param unityBuildId (Unity) the Unity build ID number. Previous name: a.ubg
+ * @param launchCount (iOS) The number of times the SDK has been launched. Previous name: a.lc
+ * @param environmentDetail (iOS) The name of the environment, i.e. dev or prod. Previous name: a.ed
+ * @param deviceManufacturer The device manufacturer. Previous name: d.dm
+ * @param deviceModel The device model. Previous name: d.do
+ * @param deviceArchitecture The CPU architecture used by the device. Previous name: d.da
+ * @param jailbroken Whether the device is rooted/jailbroken or not. Previous name: d.jb
+ * @param diskTotalCapacity The total capacity of internal storage for the whole device. Previous name: d.ms
+ * @param osType A hardcoded string representing the operating system in use. Previous name: d.os
+ * @param osVersion The human readable OS version string. Previous name: d.ov
+ * @param osAlternateType (iOS) The alternate OS type. Previous name: d.oa
+ * @param osCode (Android) The OS version code. Previous name: d.oc
+ * @param osBuild (iOS) The OS build code. Previous name: d.ob
+ * @param screenResolution The screen resolution. Previous name: d.sr
+ * @param numCores (Android) The number of CPU cores the device has. Previous name: d.nc
  */
+
 @JsonClass(generateAdapter = true)
 internal data class EnvelopeResource(
+
+    /* The app's publicly displayed version name. Previous name: a.v */
     @Json(name = "app_version")
-    val appVersion: String? = null,
+    val appVersion: kotlin.String? = null,
```

Comment:
Would be good to manually tweak the generated code to avoid the fully qualified name for these types (and other style issues). Could also go in a separate PR if you want to get it merged quickly

Reply:
I reverted that manually now. I can work on the generator after this is merged.